lib/blockchain/install.js

```diff
@@ -97,17 +97,16 @@ function plugin (crypto) {
 }
 
 function run () {
-  const ETH_CRYPTO_CODE ='ETH'
+  const ETHEREUM_CRYPTO_CODE ='ETH'
   const choices = _.map(c => {
     const checked = isInstalledSoftware(c) && isInstalledVolume(c)
     return {
       name: c.display,
       value: c.code,
       checked,
-      disabled: z => {
-        if (checked || c.cryptoCode.toString() === ETH_CRYPTO_CODE)
-          return 'Installed, use Infura option'
-      }
+      disabled: c.cryptoCode.toString() === 'ETH'
```

Comment:
Since we're using ``ETHEREUM_CRYPTO_CODE``, use it here.

Reply:
Also, I believe cryptoCode is already a string.

Reply:
copy&paste is the root of all evil.
Well spotted.